workflows/sesv2_weekly_mailer/content/70_snippets.md

```diff
@@ -0,0 +1,44 @@
+---
+skip: true
+prompt: |
+  Create an example block of YAML to describe metadata snippets. This block
+  describes one example for the EC2 StartInstances action, in Java, Python, and Rust.
```

Comment:
It's hard to know which responses were actually generated by Claude vs provided by you.

Reply:
I don't even know at this point...